pkg/operator/ceph/cluster/controller.go

```diff
@@ -216,11 +220,15 @@ func add(mgr manager.Manager, r reconcile.Reconciler, context *clusterd.Context)
 // and what is in the cephCluster.Spec
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
-func (r *ReconcileCephCluster) Reconcile(context context.Context, request reconcile.Request) (reconcile.Result, error) {
+func (r *ReconcileCephCluster) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	// workaround because the rook logging mechanism is not compatible with the controller-runtime logging interface
 	reconcileResponse, err := r.reconcile(request)
 	if err != nil {
 		logger.Errorf("failed to reconcile. %v", err)
+
+		cephCluster := &cephv1.CephCluster{}
+		_ = r.client.Get(context.TODO(), request.NamespacedName, cephCluster)
```

Comment:
Why ignoring the error?

Reply:
If there is an error here I did not want to report it. I am much worried about the original error which we got from reconcile. that's why ignoring the error.

Reply:
Ok but if there is an error I don't think we should call the recorder, right? Perhaps `r.reconcile` should return the `cephCluster` so we don't have to fetch it again?

Reply:
Done